### ISSUE TITLE
feature request

### ISSUE THREAD

**Issue by jane00** [ahmedmigo/overrideit-sketchplugin#20]
with some long text overides, the input box is too small.
I think the input box should extend automaticall accroding to the lenght of the text. But not to drag it bigger mannually everytime.

**Comment by ahmedmigo** [comment]
expand it automatically will be annoying because if you have a very long paragraph and you want to find other overrides. the text will take the full screen and it will be hard to find the other overrides.
but maybe we can work on it if you click on the field it expands automatic and if you click away it goes back

**Comment by jane00** [comment]
Yes. And now the default text input area is only 1 line height. and it will be mask by the scroll bar each time. I think you can limit the max height with auto expanding.  To do select with mouse will be much easier in multi-line input box than in a single line box.

**Comment by jane00** [comment]
And I think to do fold and unfold with the fouce behavior is too complex  and the position of other fields will changing all the time. It is unfriendly for mouse selection.
